lib/elasticsearch/document.rb

```diff
@@ -16,6 +16,17 @@ def initialize(project_model_id:, collection:)
       @model_mappings = JSON.parse(mappings_file, symbolize_names: true)[collection.to_sym][:model_fields]
     end
 
+    def thumbnail_url(provider, embed_id)
```

Comment:
This is perfect and I appreciate that you made it a separate function. Can you add another function to add the embed URL? That way we can replace [this](https://github.com/ecds/georgia-coast-atlas-frontend/blob/dev/app/components/VideoEmbed.tsx#L12-L21).